HeterogeneousCore/SonicTriton/interface/triton_utils.h

```diff
@@ -0,0 +1,29 @@
+#ifndef HeterogeneousCore_SonicTriton_triton_utils
+#define HeterogeneousCore_SonicTriton_triton_utils
+
+#include <string>
+#include <string_view>
+#include <vector>
+
+#include "request_grpc.h"
+
+namespace triton_utils {
+
+  using Error = nvidia::inferenceserver::client::Error;
+
+  template <typename T>
+  std::string printVec(const std::vector<T>& vec, const std::string& delim = ", ");
+
+  //helper to turn triton error into exception
+  void throwIfError(const Error& err, const std::string_view& msg);
```

Comment:
The `std::string_view` is cheap to copy, so it can be (and, at least implicitly, appears to be recommended) passed by value 
```suggestion
  void throwIfError(const Error& err, std::string_view msg);
```

This is my last comment. @fwyzard do you have any?